### PR TITLE
Changes of 14/16 aug discussion

### PR DESCRIPTION
**Pull request by ghost**
Hi Vijay

We have done following changes as per discussion of 14/16 Aug.

- To add version attribute for objects make it as hard coded value 1.0.0
- To add schema objects in left size of page
- To add phone no verified and email verified

Following items yet in progress, we will do it by tomorrow.

- Create link in yaml for process flow in dci_eventtype, dci_eventtype need to have mosip event like steps and link to git book
- Changes for related person of dci_person	- create related_person.yaml in dci extension

Regards
Dhananjay

### CODE REVIEW

File: src/registry/registry_core_api_v1.0.0.yaml [hunk 1]
```diff
@@ -1063,24 +1122,56 @@ components:
       $ref: schema/core/AllRecordTypes.yaml
     coreReg_EventType: 
       $ref: schema/core/EventType.yaml
-    FetchSubscriptionsRequest: 
-      $ref: schema/core/FetchSubscriptionsRequest.yaml
+    FetchSubscriptionsRequest:
+      type: object
+      properties:
+        version:
```
Comment (vvujjini) [inline src/registry/registry_core_api_v1.0.0.yaml:1127]: @ahi-dev-dc Can version be moved to respective object yaml file in registry/schema/core/*.yaml files? This comment applied to all below version attribute made in this main registry_code_api yaml file.

File: src/common/schema/Version-1.0.0.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+description: "The version format should use semantic versioning, which means dividing each version identifier into three parts: MAJOR, MINOR, and PATCH. These parts are then combined using the format 'MAJOR.MINOR.PATCH' "
+type: string
+example: '1.0.0'
```
Comment (vvujjini) [inline src/common/schema/Version-1.0.0.yaml:3]: Can we rename this file to Version_V1.0.0.yaml to keep it consistent with other similar naming convention? i.e like in msg header, callback header file.

Comment (vvujjini) [reply src/common/schema/Version-1.0.0.yaml:3]: Alternatively, why can't we just hardcode the version with 1.0.0 in the actual yaml files? So tomorrow if object definition changes a new file can be created with 1.0.1 or 1.1.0 default attribute. if we take this approach then this file is may not be required.

**Review by vvujjini — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Hi Vijay

This merge seems reverted, do you want me to raise a new pull request?

Regards
Dhananjay

**Comment by vvujjini** [comment]
Yes. I wanted to do partial merge and messed up!That will be great if you can do a new PR. Regards,Vijay On 31-Aug-2023, at 8:55 AM, ahi-dev-dc ***@***.***> wrote:﻿
Hi Vijay



This merge seems reverted, do you want me to raise a new pull request?



Regards

Dhananjay



On Wed, Aug 30, 2023 at 11:04 PM Vijay Vujjini ***@***.***>

wrote:

**Pull request merged**